### DISCUSSION TITLE
Setup run tries to access path /setup/start, which doesn't exist.

### DISCUSSION THREAD

**Discussion by lazertis** [Kovah/LinkAce#672] [Support] [answered]
Dear all,

First of all, I tried the LinkAce Demo and liked it very much, great tool! I'm very excited to use it on my own server (hosted by Hetzner as well btw :)).

Unfortunately, I get problems when trying the initial setup, which points to a non existing URL "/setup/start".

Envorinment:
- Rocky Linux 9
- Apache 2.4.53
- PHP 8.2.8 (running as php-fdm), all extensions according to setup docs installed

Done:
- Unzipped linkace 1.12.2 into a new subdir /var/www/html/linkace
- copied .env.example to .env, no further editing
- chown'ed whole directory recursively to User apache (as used by apache and php-fdm)
- ran "php artisan key:generate" -> "INFO  Application key set successfully."
- created Subdomain linkace.mydomain.de in DNS ("mydomain" is anonymized :))
- created vHost for Domain pointing (DocumentRoot) to /var/www/html/linkace/public/

When now trying "https://linkace.mydomain.de/" in Browser, it is redirecting to "https://linkace.mydomain.de/setup/start", which doesn't exist (Status 404).

There is no setup dir inside linkace tree /var/www/html/linkace nor is there any other file or directory created during this first run. access_log only point to the 404, no entry in error_log.

Do you have any idea, what went wrong here or what I can check additionally?

Thank you very much for your support! :)
Elmar

**Comment by Kovah** [comment]
Hi, it is correct that there is no setup directory. If your web server (Apache) is not loading /setup/start, please make sure that `mod_rewrite` ([guide](https://wiki.ubuntuusers.de/Apache/mod_rewrite/)) is loaded, and that the `/public/.htaccess` file is present.

**Reply by lazertis** [reply, marked as answer]
Hi, thanks for the quick answer!

Actually, rewrite is already enabled:

```
[root@gort conf.modules.d]# grep -i rewrite *
00-base.conf:LoadModule rewrite_module modules/mod_rewrite.so

[root@gort conf.modules.d]# httpd -M | grep rewrite
 rewrite_module (shared)
```
...but the failure is still on my side as I forgot to enable the directives in existing .htaccess:

```
		<Directory "/var/www/html/linkace/public/">
			AllowOverride All
		</Directory>
```
That did the job. Thanks anyway, you're rewrite hint pointed me into the right direction. :)

Greetings
Elmar
